assets/pages/paper-subscription-landing/paperSubscriptionLandingPageActions.js

```diff
@@ -36,7 +36,9 @@ const redirectToCheckout = () =>
     ) : null;
 
     if (location) {
-      sendTrackingEventsOnClick('main_cta_click', 'Paper', null)();
+      // this is annoying because we *know* state.page.plan.plan exists --------------v
```

Comment:
you could move this check higher up (it's also on `location` and have the function do nothing if it's not there